packages/shared/src/components/auth/LoginForm.tsx

```diff
@@ -61,6 +61,7 @@ function LoginForm({
         hint={hint as string}
         onChange={() => hint && setHint(null)}
         valid={!hint}
+        required
```

Comment:
Just be aware this also introduces the native browser validation, which is another task where we want to disable that.

Reply:
Oh, cool. Thanks for the note! 🚢